src/PoShLog.psd1

```diff
@@ -78,7 +78,7 @@ FunctionsToExport = 'Write-DebugLog', 'Write-ErrorLog', 'Write-FatalLog', 'Write
                'Close-Logger', 'New-LevelSwitch', 'New-Logger', 'Set-LevelSwitch', 
                'Set-Logger', 'Set-MinimumLevel', 'Start-Logger', 
                'Add-EnrichFromLogContext', 'Add-EnrichWithErrorRecord', 
-               'Add-EnrichWithProperty', 'Get-JsonFormatter', 'Convert-LogFunctions', 
+               'Add-EnrichWithProperty', 'Get-JsonFormatter', 'Get-ElasticFormatter', 'Convert-LogFunctions', 
```

Comment:
There may be pushback in building this formatter into the base module rather than an extension module: https://github.com/PoShLog/PoShLog/wiki/Extension-modules

Reply:
Sure, and I'm not against just moving it into our logging module either, it was just quicker to stand up here. I'll take a stab at that.

Reply:
It is just a formatter, not a sink, so maybe they'd be okay with it? Could be worth the PR to get their thoughts.